eslint.config.js

```diff
@@ -64,7 +64,7 @@ export default ts.config(
     },
   },
   {
-    files: ['bin/**/*', '**/*.spec.ts', 'testing/**/*', 'test/**/*'],
+    files: ['generators/**/*', 'bin/**/*', 'cli/**/*', '**/*.spec.ts', 'testing/**/*', 'test/**/*'],
```

Comment:
This looks a bug in import-x plugin.
bin/*.spec.ts/test files are not exported so we should ignore.
testing folder uses dependencies from peerDependencies so its not resolved from dependencies.

I think we should ignore yeoman-generator/yeoman-environment using https://github.com/un-ts/eslint-plugin-import-x/blob/master/docs/rules/no-unresolved.md#ignore instead

Reply:
@mshima this is what you want #27188 ?